package-res/resources/web/test/prompting/PromptPanelSpec.js

```diff
@@ -319,8 +319,10 @@ define([ 'dojo/number', 'dojo/i18n', 'common-ui/prompting/PromptPanel',
       });
 
       it("submit", function() {
+        spyOn(panel, "onSubmit");
         panel.submit();
         expect(panel.forceAutoSubmit).toBeFalsy();
+        expect(panel.onSubmit).toHaveBeenCalled();
       });
 
```

Comment:
For example,try to set some object (not function) and onSubmit should not be executed but should appear warning message.